validator/templates/validator/datasets.html

```diff
@@ -173,6 +182,25 @@
                 </ul>
             </td>
 		</tr>
+		<tr>
+			<th scope="row">ERA5</th>
+			<td>ERA5 is a global reanalysis product provided by ECMWF. It provides global, sub-daily simulations of variables for land, 
+				atmosphere and ocean waves. The downloaded original 6-hourly (starting at 0:00 UTC) images of ERA5 Volumetric Soil Moisture 
+				with a spatial resolution of 0.25° Lat. * 0.25° Lon were converted into a time series format via daily averages. 
+				ERA5 covers the period from 1979 to 2-3 months before the present. The version number used in QA4SM indicates the date 
```

Comment:
"ERA5 covers the period from 1979 to 2-3 months before the present" ... that might be confusing because the data in QA4SM is not updated regularly...just say that it starts in 1979

Reply:
updated